src/main/kotlin/org/rust/lang/core/psi/ext/RsMetaItem.kt

```diff
@@ -32,3 +33,33 @@ val RsMetaItem.hasEq: Boolean get() = greenStub?.hasEq ?: (eq != null)
 
 fun RsMetaItem.resolveToDerivedTrait(): RsTraitItem? =
     path?.reference?.resolve() as? RsTraitItem
+
+/**
+ * In the case of `#[foo(bar)]`, the `foo(bar)` meta item is considered "root", but `bar` is not.
+ * In the case of `#[cfg_attr(windows, foo(bar))]`, the `foo(bar)` is also considered "root" meta item
+ * because after `cfg_attr` expanding the `foo(bar)` will turn into `#[foo(bar)]`.
+ * This also applied to nested `cfg_attr`s, e.g. `#[cfg_attr(windows, cfg_attr(foobar, foo(bar)))]`
+ */
+val RsMetaItem.isRootMetaItem: Boolean
+    get() = parent is RsAttr || isCfgAttrBody
+
+/**
+ * ```
+ * #[cfg_attr(condition, attr)]
+ *                     //^
+ * ```
+ */
+private val RsMetaItem.isCfgAttrBody: Boolean
+    get() {
+        val parent = parent as? RsMetaItemArgs ?: return false
+        val parentMetaItem = parent.parent as? RsMetaItem ?: return false
+
+        if (!parentMetaItem.isCfgAttrMetaItem) return false
+
+        val siblingMetaItems = parent.metaItemList
+        return siblingMetaItems.size == 2 && siblingMetaItems.lastOrNull() == this
```

Comment:
Note, you can pass more than one attributes into `cfg_attr`. For example,  `#![cfg_attr(unix, allow(), warn())]` is valid in Rust

Reply:
Wow